src/test/scala/mesosphere/marathon/core/storage/store/impl/zk/RichCuratorFrameworkTest.scala

```diff
@@ -5,7 +5,7 @@ import java.util.UUID
 
 import akka.util.ByteString
 import mesosphere.UnitTest
-import mesosphere.marathon.integration.setup.ZookeeperServerTest
+import mesosphere.marathon.util.ZookeeperServerTest
```

Comment:
I don't understand this change. How come `ZookeeperServerTest` has a different package now? I cannot find the related change.

Reply:
Because ZookeeperServerTest was used by multiple unit tests